pinot-broker/src/main/java/org/apache/pinot/broker/routing/instanceselector/BaseInstanceSelector.java

```diff
@@ -50,9 +49,8 @@ abstract class BaseInstanceSelector implements InstanceSelector {
   private static final Logger LOGGER = LoggerFactory.getLogger(BaseInstanceSelector.class);
 
   // To prevent int overflow, reset the request id once it reaches this value
-  private static final int MAX_REQUEST_ID = 1_000_000_000;
+  private static final long MAX_REQUEST_ID = 1_000_000_000;
```

Comment:
is this change needed?

Reply:
Technically no, but since I was taking `requestId % MAX_REQUEST_ID` in `select` and `requestId` is a long there thought of explicitly making it a long.